.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ckan-version: [2.9, 2.9-py2, 2.8]
+        ckan-version: ["2.10", 2.9, 2.9-py2, 2.8]
```

Comment:
Why is 2.10 quoted when the other versions aren't?

Reply:
Because it ends with a zero and will therefore be interpreted as "2.1" if it's unquoted.